src/ai/backend/agent/docker/metadata/server.py

```diff
@@ -56,7 +57,11 @@ async def container_resolver_middleware(
         return web.Response(status=HTTPStatus.FORBIDDEN)
     async with closing_async(Docker()) as docker:
         containers = await docker.containers.list(
-            filters='{"label":["ai.backend.kernel-id"],"network":["bridge"],"status":["running"]}',
+            filters=(
+                '{"label":["'
+                + LabelName.KERNEL_ID.value
+                + '"],"network":["bridge"],"status":["running"]}'
```

Comment:
JSON should be converted to JSON after creating the dataclass rather than applying it as a string.